src/Content/Migration/Base.php

```diff
@@ -409,8 +409,10 @@ public function addComponentEntry($name, $option=null, $enabled=1, $params='', $
 			$query .= " VALUES ('{$name}', 'option={$option}', 0, 0, 'option={$option}', '{$name}', '{$option}', {$ordering}, '', 0, ".$this->baseDb->quote($params).", {$enabled})";
 			$this->baseDb->setQuery($query);
 			$this->baseDb->query();
+
+			return true;
 		}
-		else
+		elseif ($this->baseDb->tableExists('#__extensions'))
```

Comment:
> Note: Note that elseif and else if will only be considered exactly the same when using curly brackets as in the above example. When using a colon to define your if/elseif conditions, you must not separate else if into two words, or PHP will fail with a parse error.

Stylistically, should we stick to combining to elseif in this case?

Reply:
Yes, we should settle on the format that works best in both cases.